source/administration/upgrade.md

```diff
@@ -263,22 +203,31 @@ Private cloud enterprise communications server.
 
 - Mattermost Enterprise Edition v3.6.0 - [View Changelog](https://docs.mattermost.com/administration/changelog.html#release-v3-6-0) - [Download](https://releases.mattermost.com/3.6.0/mattermost-3.6.0-linux-amd64.tar.gz)
   - `https://releases.mattermost.com/3.6.0/mattermost-3.6.0-linux-amd64.tar.gz`
+  - Checksum: `c58110460c33dbe3d365eaea25ae0cf2e4c529fd81c70b969f8c5611d467bff6`
 - Mattermost Enterprise Edition v3.5.1 - [View Changelog](https://docs.mattermost.com/administration/changelog.html#release-v3-5-1) - [Download](https://releases.mattermost.com/3.5.1/mattermost-3.5.1-linux-amd64.tar.gz)
```

Comment:
What kind of checksum? At first glance looks like SHA-256, but I could be wrong.